packages/api/bin/cli.js

```diff
@@ -71,9 +71,11 @@ program
 
 program
   .command('serve')
+  .option('--stackName <stackName>', 'stackname to serve', 'localrun')
```

Comment:
I think it might be easier to manage the default value, `localrun`, exclusively in the `bin/serve.js` file. I'll explain below